content/knowledge-others/install-unity-version.md

```diff
@@ -99,32 +99,93 @@ Use the Unity version you installed on the machine:
   scripts:
     - name: Build the Unity app
       script: |  
-        cmd.exe /c "$env:UNITY_VERSION_BIN" -batchmode -quit -logFile -projectPath . -executeMethod BuildScript.$env:BUILD_SCRIPT -nographics
+        cmd.exe /c "$env:UNITY_VERSION_BIN" -batchmode -quit -logFile -projectPath . -executeMethod BuildScript.$env:BUILD_SCRIPT -nographics -buildTarget Android   
 {{< /highlight >}}
 {{< /tab >}}
 
 {{< /tabpane >}}
 
+{{<notebox>}}
+When building with the new Unity version, it's better to add the **-buildTarget** argument to set the build target to `android` or `ios`.
+{{</notebox>}}
 ## Caching
-You can cache the newly installed version on Unity to avoid installing it in every build, read more about caching [here](https://docs.codemagic.io/yaml/yaml-getting-started/#cache).
+Instead of waiting from 5 to 7 minutes in every build to download and install your Unity editor, you can cache it using your external storage tool like AWS S3 or GCP.
+
+In the following example we are going to use AWS S3 to store our cached files.
+
+#### Using AWS S3
+In order to use AWS S3, you need to configure your access credentials in Codemagic. You can follow the [instructions](https://aws.amazon.com/getting-started/hands-on/backup-to-s3-cli/) provided by Amazon to create your account and get the necessary details.
+
+1. Open your Codemagic app settings, and go to the **Environment variables** tab.
+2. Enter the desired **_Variable name_**, e.g. `AWS_ACCESS_KEY_ID`.
+3. Enter the required value as **_Variable value_**.
+4. Enter the variable group name, e.g. **_aws_credentials_**. Click the button to create the group.
+5. Make sure the **Secure** option is selected.
+6. Click the **Add** button to add the variable.
+7. Repeat the process to also add the `AWS_SECRET_ACCESS_KEY` variable.
+
+8. Import the  **_aws_credentials_** group and install the **awscli** using this script.
 
-{{< tabpane >}}
-{{% tab header="Mac" %}}
 {{< highlight yaml "style=paraiso-dark">}}
-    cache:
-      cache_paths:
-        - /Applications/Unity/Hub/Editor/${UNITY_VERSION}
+environment:
+  groups:
+    - aws_credentials
+scripts:
+  - name: Install awscli
+    script: | 
+      sudo pip3 install awscli --upgrade
```

Comment:
AWS cli 2.8.9 is pre-installed, so shouldn't need to be upgraded for S3 uploads.